tests/resources/poor_config.yaml

```diff
@@ -0,0 +1,32 @@
+
+# NOTE THIS CONFIG IS HERE TO BE USED TO NOT HAVE THE REQUIRED KEYS IN THE TESTS!
```

Comment:
Could "incomplete_config.yml" be a clearer name? Suggestions for comment text that could clarify things a bit: "This config is a partial copy of checkQC/default_config/config.yaml, with required keys removed in order to trigger errors. For testing only."

Reply:
Good point. I'll fix that.